src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/disk_encryption.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 import uuid
+import os
 from msrestazure.tools import parse_resource_id
```

Comment:
@yugangw-msft 
My local build seems to be breaking on this import. Any idea what's up?

Reply:
Nevermind, it was a local build issue